includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -475,4 +497,51 @@ private function formatLocalNavData( array $items, int $nestingLevel ): array {
 	private function getSpecialPageURL( $name ): string {
 		return SpecialPage::getTitleFor( $name )->getLocalURL();
 	}
+
+	private function getFandomStoreDataFromCache() {
+		global $wgMemc, $wgCityId;
+
+		$key = DesignSystemApiController::MEMC_PREFIX_FANDOM_STORE;
+		$memcKey = "intentX:$key:$wgCityId";
```

Comment:
Instead of manually formatting the key you should use `$wgMemc->makeKey( 'intentX, DesignSystemApiController::MEMC_PREFIX_FANDOM_STORE )`

Reply:
I was looking for something like that. `wfMemcKey` kept adding a prefix I didn't want. I actually just used `substr` to get the value I was looking for without the prefix.

Reply:
I think that will add the community id to the key which we don't want because when we call from `community` it will cause an issue.